src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -54,6 +54,14 @@
 @Configuration
 public class Utils {
 
+	public static final String ROLE_OWNER = "Owner";
+	public static final String ROLE_ORGANIZER = "Organizer";
+	public static final String ROLE_MEMBER = "Member";
+	public static final String ROLE_OBSERVER = "Observer";
+	public static final String ROLE_MAINTAINER = "maintainer";
+	public static final String ROLE_INSTRUCTOR = "Instructor";
+	public static final String ROLE_STUDENT = "student";
```

Comment:
In fact, this is the actual data values in CTools database. We will not change it.